source/doc_versions.py

```diff
@@ -1,9 +1,9 @@
 # The API version strings to display in the documentation
 # COMMON_VERSION is the latest version of any one API, and is considered the overall API version
-COMMON_VERSION = '2016-06-12'
+COMMON_VERSION = '2016-07-01'
```

Comment:
I agree this does impact the public facing API (response).

> We don't record the error structure on the API doc though so there are no changes made to the documentation.
Shouldn't we document them? Seems like a miss from before and now would be a great time to add some documentation of how errors are returned for an invalid campaign (with IVR tree)